inhouse_bot/common_utils/lol_api/pyotconf.py

```diff
@@ -0,0 +1,34 @@
+import os
+from pyot.conf.model import activate_model, ModelConf
+from pyot.conf.pipeline import activate_pipeline, PipelineConf
+
+
+@activate_model("lol")
```

Comment:
can consider putting these values in docker compose

Reply:
true, but for now the only region is NA1